README.md

```diff
@@ -9,6 +9,12 @@ Most documentation for the service can be found on the
 Some app-specific technical documentation can be found in the [docs](docs)
 directory.
 
+### dxw support
```

Comment:
Yeah, it would have meant I could do little things like writing the best place to ask for help, but you might be right. Anyway, I've changed it to make it generic.